### DISCUSSION TITLE
How do I update the value of one field according to another?

### DISCUSSION THREAD

**Discussion by sajaddp** [laravel/nova-issues#4492] [Q&A]
Hello
I have two tables `A` and `B`.
Table `A` has three columns called `b_id`, `hour`, and `amount`.
Table `B` also has a `user_id` and `salary` column.

The relationship between the two tables is also well defined.

When the user creates an `A`, I want the `amount` to be filled in automatically according to the following relation:
`amount` = `hour` * `b.salary`

How do I do this?

**Comment by anditsung** [comment]
using observer or event should solve this
